src/components/Disputes/panels/AppealPanel.js

```diff
@@ -58,66 +58,76 @@ const AppealPanel = React.memo(function AppealPanel({
   // Reqiured deposits for appealing and confirming appeal are different
   const requiredDeposit = confirm ? confirmAppealDeposit : appealDeposit
 
-  // If appealing => options are the opossed of the wining outcome
-  // If confirming appeal => options are the opossed of the appealed ruling
   const { vote, appeal } = getDisputeLastRound(dispute)
-
-  // Cases where a confirm appeal is done, the next round is created (with no appeal) and the panel hasn't closed yet
-  if (confirm && !appeal) return null
-
   const { winningOutcome } = vote || {}
+
+  // If appealing => options are the opossed of the wining outcome
+  // If confirming appeal => options are the opossed of the appealed ruling
   const appealOptions = getAppealRulingOptions(
     confirm ? appeal.appealedRuling : winningOutcome
   )
 
-  // check if connected account has the minimum required deposit to be able to appeal
-  const canAppeal = feeBalance.gte(requiredDeposit)
-
-  // Form validation
-  const validateForm = outcome => {
-    if (outcome === -1) {
-      const error = 'You must select an outcome'
-      setSelectedOutcome(outcome => ({ ...outcome, error }))
-      return true
-    }
-
-    return false
-  }
-
   // For submission
-  const handleAppeal = async event => {
-    try {
+  const handleAppeal = useCallback(
```

Comment:
Nitpick: might be nice to move the other `useCallback()` function (`handleOutcomeSelected()`) close to this function; in other places we've taken the habit of grouping "alike" hooks together, since they usually do similar things:

1. `useState()` / custom state hooks: these provide state and "context" for logical operations later in the component
1. `useMemo()`, `useRef()`: these are usually derived from props or state, and continue providing state and context
1. `useCallback()`: these are usually reactive handlers passed into props
1. `useEffect()`: these usually derive from the above, and can be viewed as "automatic triggers" (similar in concept to render, which is always last; it _reacts_ based on all the above changing).